source/conf.py

```diff
@@ -0,0 +1,85 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+
+import os
+import sys
+sys.path.insert(0, os.path.abspath('../api/app'))
+
+# -- Project information -----------------------------------------------------
+
+project = 'EAK-Copilot'
+copyright = '2024, CdC-SI'
+author = 'CdC-SI'
+
+# The full version, including alpha/beta/rc tags
+release = '0.1.0'
+
+
+# -- General configuration ---------------------------------------------------
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    'sphinx.ext.autodoc',
+    'sphinx.ext.autosummary',
+]
+autodoc_mock_imports = ["fastapi"]
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files.
+# This pattern also affects html_static_path and html_extra_path.
+exclude_patterns = []
+
+
+# -- Options for HTML output -------------------------------------------------
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+#
+html_theme = 'furo'
+html_theme_options = {
+    "source_repository": "https://github.com/CdC-SI/eak-copilot",
+    "source_branch": "feature/161-investigating-sphinx/",
```

Comment:
maybe after merge you have to chance the source_branch to main?

Reply:
Yes exactly.

Also, can I edit directly the sphinx related files in the main branch after merging (without creating new branch and pull request) ? Those changes would be unrelated to the copilot code so it shouldn't harm.

Reply:
Also, what should i do with the MKDocs and Docusaurus branches? Should I delete them? @tabee